integration_test/ops_agent_test/main_test.go

```diff
@@ -909,14 +937,12 @@ func TestKillChildJobsWhenPluginServerProcessTerminates(t *testing.T) {
 
 func TestCustomLogFormat(t *testing.T) {
 	t.Parallel()
-	RunForEachLoggingSubagent(t, func(t *testing.T, otel bool) {
+	RunForEachImageAndLoggingSubagent(t, func(t *testing.T, imageSpec string, otel bool) {
```

Comment:
This would change to:
```
RunForEachFeatureFlag(t, []string{"otel"}, func(t *testing.T, feature string) {

```
you can check for your feature by checking the value of `feature`